pubsub/tests/unit/pubsub_v1/subscriber/test_message.py

```diff
@@ -53,53 +54,52 @@ def test_publish_time():
 def test_ack():
     msg = create_message(b'foo', ack_id='bogus_ack_id')
     with mock.patch.object(msg._request_queue, 'put') as put:
-        with mock.patch.object(message.Message, 'drop') as drop:
-            msg.ack()
-            put.assert_called_once_with(('ack', {
-                'ack_id': 'bogus_ack_id',
-                'byte_size': 25,
-                'time_to_ack': mock.ANY,
-            }))
+        msg.ack()
+        put.assert_called_once_with(base.AckRequest(
```

Comment:
These tests are actually less precise now because of the generous `__eq__` implementation in `namedtuple`.

Recommend changing all of these to:

```python
_, args, _ = put.mock_calls[0]
assert isinstance(args[0], base.AckRequest)  # namedtuple equality is permissive.
assert args[0] == base.AckRequest(ack_id='bogus_ack_id', byte_size=25, time_to_ack=mock.ANY)
```

Reply:
Done in 33947c285